frontend/test/parsing/testParseArrayDomainRange.cpp

```diff
@@ -159,7 +159,6 @@ int main() {
   testRange(p, "testRange4.chpl", "..<", true, true);
   testRange(p, "testRange5.chpl", "..<", false, true);
 
-  testArrayDomain(p, "testArray0.chpl", true, 1, false);
```

Comment:
It would probably be nice to reindex these `testArray#` test names from 0.
Additionally, it won't let me tag it as it's not close to a modified line, but there's a [TODO on `testArrayDomain`](https://github.com/chapel-lang/chapel/blob/4897a04842c0b07c833dc80115213967e0503440/frontend/test/parsing/testParseArrayDomainRange.cpp#L82) that I think can be removed with this PR.

Reply:
And people wonder why I'm not a fan of unit testing!  :D

Reply:
> there's a [TODO on testArrayDomain](https://github.com/chapel-lang/chapel/blob/4897a04842c0b07c833dc80115213967e0503440/frontend/test/parsing/testParseArrayDomainRange.cpp#L82) that I think can be removed with this PR.

Maybe I'm not understanding the TODO, but it doesn't seem like I've done anything in this PR that puts us in a better spot w.r.t. the unit testing that's being done?  Or are you saying that you think the TODO could've been removed before now, so while I'm here I might as well remove it?

Reply:
Aha, @dlongnecke-cray set me straight!